### DISCUSSION TITLE
env_file does nothing and I'm lost as to how to troubleshoot, is there a debug mode?

### DISCUSSION THREAD

**Discussion by red8888** [pydantic/pydantic#8601] [Question]
I put a mysettings.env in my current working dir and configured it:

```
    model_config = SettingsConfigDict(
        env_prefix="my_app_",
        env_file=[
            "mysettings.env",
        ],
    )
```

I know the settings are working because exporting a var **MY_APP_FIELD** works, but it runs as if mysettings.env doesn't exist.

What do I do to see where its trying to load my config files? Is there any kind of debug mode here because I could see this becoming a real problem- ie figuring out which files were and were not loaded.

**Comment by red8888** [comment]
Think I found a bug: https://github.com/pydantic/pydantic/issues/8602

Setting env_prefix seems to break env_file! If I remove env_prefix it loads the file- bug?

Also, I still have yet to find a debug mode or something where I can ask pydantic to log everything its doing (ie looking for env "mysettings.env", could not find file, etc, etc)

**Comment by hramezani** [comment]
Closing in favor of the reported bug
